api-reference/beta/api/directory_get.md

```diff
@@ -0,0 +1,74 @@
+# Get deletedItem
+
+Retrieve the properties of an item in [deleted items](../resources/directory.md).
+
+### Prerequisites
+One of the following **scopes** are required to execute this API:
+* For users: *User.ReadBasic.All; User.Read.All; User.ReadWrite.All; Directory.Read.All; Directory.ReadWrite.All; Directory.AccessAsUser.All*
+* For groups: *Group.Read.All or Group.ReadWrite.All; Directory.Read.All; Directory.ReadWrite.All; Directory.AccessAsUser.All*
+
+### HTTP request
+<!-- { "blockType": "ignored" } -->
+```http
+GET /directory/deletedItems/Microsoft.Graph.Group/{id}
+GET /directory/deletedItems/Microsoft.Graph.User/{id}
+```
+
+This API currently supports retrieving object types of groups (Microsoft.Graph.Group) or users (Microsoft.Graph.User) from deleted items. The type is specified as part of the URI.
```

Comment:
I think that's fine.  I think you can further add that calling GET /directory/deletedItems is not supported without specifying a cast.  I think GET /directory/deletedItems/{id} should be supported.